src/main.c

```diff
@@ -3409,6 +3516,8 @@ static int on_config_onoff(h2o_configurator_command_t *cmd, yoml_t *node, int *s
     return 0;
 }
 
+#include "handler/configurator/busypoll.c"
```

Comment:
As stated in my comment on CMakeLists.txt, I think we do not want to start adding `lib` as an include directory.

Considering that lib/handler/configurator/busypoll.c defines functions that change the variable `conf`, which is a static variable of src/main.c, I think it might make sense to define those callbacks inside src/main.c.

The alternative would be to do something like what src/ssl.c does. I.e.,
* move lib/hander/configurator/busypoll.c to src/,
* in src/busypoll.c, define a global variable dedicated to busypoll settings,
* add declarations of that global and the functions to src/standalone.h, and
* call them from src/main.c.